luda-editor/new-client/src/episode_editor/scene_list.rs

```diff
@@ -53,9 +63,15 @@ impl Component for SceneListCell<'_> {
         3.
         */
 
-        // for sprite in scene.sprites {
-        //     sprite.sprite_id
-        // }
-        todo!()
+        ctx.compose(|ctx| {
+            ctx.translate(Xy::single(4.px()))
+                .add(typography::body::left_top(index.to_string(), Color::WHITE));
+        });
+
+        for scene_sprite in &scene.scene_sprites {
+            render_psd_sprite(ctx, scene_sprite, wh);
```

Comment:
is it ok not to use ctx compose with key for this?

Reply:
Seems needed. I will fix in this pr